### DISCUSSION TITLE
Why can't I see dataview output?

### DISCUSSION THREAD

**Discussion by iomadh** [oleeskild/obsidian-digital-garden#290] [Q&A] [answered]
Hi, just installed the digital garden plugin and deployed to Netlify ( I followed the instructions from the docs and didn't get any errors).  I updated the template and can publish fine.  Except, dataview outputs only display as text, e.g.

![image](https://user-images.githubusercontent.com/7075466/234369436-ed203f61-f877-4997-9499-1f7b4b52188a.png)

Have a missed a step?

**Comment by iomadh** [comment, marked as answer]
I worked it out

my dataviews all have a space in the heading 

` ``` dataview `

whilst the dataview plugin renders this correctly, the digitial garden plugin does not
Once I changed to

` ```dataview `

DG plugin renders correctly.  Hope this helps someone else in the future

Thanks for the excellent plugin

**Reply by nottrobin** [reply]
OMG thank you! This was so frustrating!

**Comment by rokville** [comment]
as an fyi, this appears to be an issue I was experiencing with inline(?) dataview queries too.  `= this.file.name` failed, but `=this.file.name` works.
